layer/trace_layer.cpp

```diff
@@ -53,6 +53,58 @@ const VkLayerProperties kLayerProps = {
 
 const std::vector<VkExtensionProperties> kDeviceExtensionProps = { VkExtensionProperties{ "VK_EXT_tooling_info", 1 } };
 
+/// An alphabetical list of device extensions which we do not report upstream if
+/// other layers or ICDs expose them to us.
+const char* const kDeviceExtensionBlacklist[] = {
```

Comment:
I know this is going to sound PC but would you change "Blacklist" to "Blocklist" in your change?

Reply:
I've done something equivalent.